app/mailers/concerns/contact_mailer.rb

```diff
@@ -0,0 +1,35 @@
+module ContactMailer
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :start_contact_log
+    after_action :finish_contact_log
+  end
+
+  private
+
+    def start_contact_log
+      # TODO: is there a better way to handle this content required field?
```

Comment:
I tried playing around with different attempts at `before_action`, `around_action` and `after_action` but gave up without digging in too much and just did this workaround. I don't love adding a dummy value for content and then updating the other attributes later but it does seem to work. Perhaps there is a better way.

Reply:
duh, done